src/7bb60_len_41b0.c

```diff
@@ -1069,7 +1069,7 @@ void collision_lateral_peach(void) {
     playerStatus->position.z = z;
 
     // If there was a climbable step in this direction, but no wall, we can climb up it
-    if (climbableStep != 0 &&
+    if (climbableStep != FALSE &&
```

Comment:
ah, one more thing - this can just be `if (climbableStep &&`

Reply:
Wasn't sure it would match but indeed it does